include/psen_scan_v2/tenth_of_degree.h

```diff
@@ -49,7 +54,7 @@ class TenthOfDegree
     return *this;
   }
 
-  constexpr TenthOfDegree& operator*(const unsigned int& rhs)
+  constexpr TenthOfDegree& operator*(const int& rhs)
```

Comment:
What happens then this overflows?

Reply:
I don't know. Do you suggest that we detect overflow of integer arithmetic?